example/index.php

```diff
@@ -21,7 +21,7 @@
         $config['api_hostname'],
         $config['redirect_uri'],
         true,
-        $config['http_proxy'],
+        $config['http_proxy'] ?? null,
```

Comment:
Nit: what is this line accomplishing? My understanding of `??` is that it's a binary operator that returns the right value if the left value is null but here the right value is also null. Was this added to make a tool happy?

Reply:
Exactly, this doesn't change any behavior, but PHP8 now warns if you don't explicitly handle nulls.